license-header-check/action.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2024, NVIDIA CORPORATION.
+# Copyright (c) 2024-2025, NVIDIA CORPORATION.
```

Comment:
discuss offline, lets try refactor this to force fetch upstream base-branch and fea-branch HEAD commits, and then compare them instead of using `github.event.pull_request.base.sha vs github.event.pull_request.head.sha` which could be out of date. thanks